src/test/java/com/cloudbees/jenkins/support/SupportTestUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.cloudbees.jenkins.support;
+
+import com.cloudbees.jenkins.support.api.Component;
+import com.cloudbees.jenkins.support.api.Container;
+import com.cloudbees.jenkins.support.api.Content;
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+/**
+ * User: schristou88
```

Comment:
If you create Javadocs, please add information details there. Date and Time are useless, use <code>@since</code> to refer the plugin version, where the fix will be released